packages/client-core/src/admin/common/Table.tsx

```diff
@@ -49,20 +49,16 @@ export interface ITableHeadCell {
 
 interface TableHeadProps {
   onRequestSort: (property: string | number, order: FeathersOrder) => void
-  order: FeathersOrder
-  orderBy: string | number
+  order: ImmutableObject<Record<string, FeathersOrder>>
   columns: ITableHeadCell[]
 }
 
-const TableHead = ({ order, orderBy, onRequestSort, columns }: TableHeadProps) => {
+const TableHead = ({ order, onRequestSort, columns }: TableHeadProps) => {
   const SortArrow = ({ columnId }: { columnId: string | number }) => {
-    if (columnId === orderBy) {
-      if (order === 1) {
-        return <HiArrowSmallUp onClick={() => onRequestSort(columnId, -1)} />
-      }
-      return <HiArrowSmallDown onClick={() => onRequestSort(columnId, 1)} />
+    if (columnId in order && order[columnId] === 1) {
+      return <HiArrowSmallUp onClick={() => onRequestSort(columnId, -1)} />
     }
-    return <HiArrowSmallUp className={'opacity-0 hover:opacity-50'} />
```

Comment:
this statement was there to let know the user that the column was sortable
only 1 column can be sorted at a time

check the below image

![image](https://github.com/user-attachments/assets/bf3c7392-117e-4c73-98f7-aee8a9d9fa42)

in the above image, the user might get the wrong indication that the date column is sorted but only the name column is sorted

Reply:
![image](https://github.com/user-attachments/assets/d596c5cb-ba0a-40c5-8d1d-1fad484eabf4)
Added different icon for unsorted columns, but sortable.